### DISCUSSION TITLE
What is the minimal requirement to run maddy?

### DISCUSSION THREAD

**Discussion by hao-lee** [emersion/maddy#359] [General]
Mailu needs at least 1G memory. What is the minimal system requirement of maddy? Thanks!

**Comment by foxcpp** [comment]
To guarantee smooth operation with default config, I recommend making at least 800 MB available to the server.
Most of the time maddy will use much less (around 200 MB).

**Reply by hao-lee** [reply]
Thanks. This is awesome!
